Fixes warning:
../bmalloc/bmalloc/Gigacage.h: In function ‘Gigacage::BasePtrs& Gigacage::basePtrs()’:
../bmalloc/bmalloc/Gigacage.h:136:59: warning: cast from ‘char*’ to ‘Gigacage::BasePtrs*’ increases required alignment of target type [-Wcast-align]
     return *reinterpret_cast<BasePtrs*>(g_gigacageBasePtrs);

diff --git a/Source/bmalloc/bmalloc/Gigacage.h b/Source/bmalloc/bmalloc/Gigacage.h
index bef3af5d53a..60bba03d65e 100644
--- a/Source/bmalloc/bmalloc/Gigacage.h
+++ b/Source/bmalloc/bmalloc/Gigacage.h
@@ -133,7 +133,7 @@ BINLINE void*& basePtr(BasePtrs& basePtrs, Kind kind)
 
 BINLINE BasePtrs& basePtrs()
 {
-    return *reinterpret_cast<BasePtrs*>(g_gigacageBasePtrs);
+    return *reinterpret_cast<BasePtrs*>(static_cast<void*>(g_gigacageBasePtrs));
 }
 
 BINLINE void*& basePtr(Kind kind)
